docs/notebooks/zero_inflated_regression.ipynb

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
<p>This is a little bit ambiguous: "the probability of the Poisson process, i.e., the probability of non-zero data, increases"</p><p> </p><p>It is correct to say the probability of the Poisson process increases, but that doesn't mean we have no zeroes, since zeroes are also possible under the Poisson distribution.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<blockquote>&nbsp;but that doesn't mean we have no zeroes, since zeroes are also possible under the Poisson distribution.</blockquote><p> </p><p>Yeah, that's why I added the "probability of non-zero data" since it implies that zeros are still plausible. I wanted to add an interpretation of the statement "the probability of the Poisson process" for the users. For some it may be obvious, but for others not. </p>